podman/domain/images_build.py

```diff
@@ -204,8 +204,16 @@ def _render_params(kwargs) -> dict[str, list[Any]]:
         if "labels" in kwargs:
             params["labels"] = json.dumps(kwargs.get("labels"))
 
-        if params["dockerfile"] is None:
-            params["dockerfile"] = f".containerfile.{random.getrandbits(160):x}"
+        def default(value, def_value):
+            return def_value if value is None else value
```

Comment:
having a function for this is overkill, parameters can be defined as default in kwargs calls.

consider using `kwargs.get(value, default)` like [here](https://github.com/containers/podman-py/blob/main/podman/domain/containers.py#L436-L437)

Reply:
I did not know of this. Makes more sense

Reply:
> Why do so many functions in this project use kwargs

Historical reasons probably?

@jwhonce might give us a good answer

Reply:
@fulminemizzega kwargs was a good escape hatch for podman-py to ease the migration for scripts from docker-py. It allowed developers access to podman features without having to port their whole scripts. From there I suspect there are areas where new development followed the old form even if it didn't make as much sense. 

The flexibility of args / kwargs have always been very Pythonic and pre-date type hinting.